queries.sql

```diff
@@ -30,3 +30,76 @@ select * from animals where not name='Gabumon';
 /* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/* Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before tranasction. */
+
+begin;
+update animals set species='unspecified';
+rollback;
+
```

Comment:
Please add queries before and after rollback to verify that the change was made.